module/surveys/helper.js

```diff
@@ -1413,6 +1415,172 @@ module.exports = class SurveysHelper {
     });
   }
 
+  /**
+   * List of surveys based on UserId.
+   * @method
+   * @name listSurveyByUserId
+   * @param {String} requestUserId - id of the user to fetch docmunet.
+   * @param {String} queryType -     type of collection to query whether its survey or obs
+   * @returns {JSON} List of surveys.
+   */
+
+  static listSurveyByUserId(requestUserId, queryType = "") {
+    return new Promise(async (resolve, reject) => {
+      try {
+
+        let surveyMatchQuery ={
+          $match:{
+             createdBy: requestUserId 
+
+          }
+        }
+        let surveyDocument = await this.getAggregate(
```

Comment:
for big number you need only number that can done with count document only @praveenKDass

Reply:
Added count document @ankit1shahu